packages/x-components/tests/e2e/cucumber/history-queries.feature

```diff
@@ -81,7 +81,8 @@ Feature: History queries component
     And   a "<list>" of queries already searched
     When  search-input is focused
     And   history query number <historyQueryItem> is clicked
-    And   clear search button is pressed
+    Then  related results are displayed
+    When  clear search button is pressed
```

Comment:
Is this issue related with firefox doing things too fast once again? What I saw after making it fail like it was before is that the query is properly cleared from the input, but not from other modules.

This is what Cypress saw
![image](https://user-images.githubusercontent.com/68222542/157229792-4cc53e73-4753-4710-8f06-7a2db5f54d23.png)

But when you clear the query, either by using the clear button or the delete key, if I remember correctly, every module but the next queries should reset its state, which is not happening in the test.
I think this might happen because of the asynchronicity of certain events, and how FireFox handles the task queue, but I would need to dig deeper into this.